cpp/ql/test/library-tests/dataflow/security-taint/tainted_diff.expected

```diff
@@ -8,3 +8,8 @@
 | test.cpp:68:28:68:33 | call to getenv | test.cpp:69:10:69:13 | copy | AST only |
 | test.cpp:68:28:68:33 | call to getenv | test.cpp:70:12:70:15 | copy | AST only |
 | test.cpp:68:28:68:33 | call to getenv | test.cpp:71:12:71:15 | copy | AST only |
+| test.cpp:87:12:87:15 | call to gets | test.cpp:87:2:87:8 | pointer | AST only |
+| test.cpp:87:17:87:22 | buffer | test.cpp:84:7:84:12 | buffer | AST only |
```

Comment:
Are these useful results we need to replicate in `DefaultTaintTracking::adjustedSink`?

Reply:
The second one can be added by allowing `Variable`s to be a taint sink in defaulttainttracking.  I'm not sure whether this is the right fix or should be restricted in some ways (defaulttainttracking doesn't appear to be super consistent on the matter?), and it has effects on a couple of tests elsewhere, so I might make it as a separate PR but I'm not keen to add it as a fix to this one.

The first line indicates that taint isn't flowing right-to-left in `pointer = gets(buffer);`.  This is very strange - dataflow itself should be handling this case, and I haven't yet figured out why it isn't.